src/Symfony/Component/DomCrawler/Form.php

```diff
@@ -172,6 +172,16 @@ public function getPhpFiles()
             if (!empty($qs)) {
                 parse_str($qs, $expandedValue);
                 $varName = substr($name, 0, strlen(key($expandedValue)));
+
+                array_walk_recursive(
+                    $expandedValue,
+                    function (&$value, $key) {
+                        if (is_numeric($value) && in_array($key, ['size', 'error'], true)) {
```

Comment:
is_numeric works with floats also, so better use ctype_digit here if we handle only integers, isn't it?
also, please use `array(...)` instead of `[...]`, or better: `('size' === $key || 'error' === $key)`